wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java

```diff
@@ -192,11 +192,8 @@ public static boolean isFailedState(String state) throws Exception
             WuStateNameMap.put("RUNNING", WUState.RUNNING);
     }
 
-    public static WUState translateWUState(String state) throws Exception
+    public static WUState translateWUState(String state)
     {
-        if (WuStateNameMap.size() <= 0)
```

Comment:
ok, but the throws was removed, which could affect code that called this function, please make sure that callers of this function (and callers of the caller) are not throwing this exception anymore.